src/mrgw.sh

```diff
@@ -11,7 +11,7 @@ port=$2
 
 shift 2
 
-run_root=$script_root/run/$name
+run_root=run/$name
```

Comment:
that's correct. though in the cmake case, running mrgw.sh from anywhere but the cmake build directory will fail because it doesn't know how the find the binaries. in the automake case, I assume that it will fail if it doesn't see a ./run directory from mstart